src/index.html

```diff
@@ -17,7 +17,7 @@
 -->
 <html lang="en">
 <head>
-    <title translatable="yes">Cockpit Starter Kit</title>
+    <title translate="yes">Cockpit Starter Kit</title>
```

Comment:
Isn't just `<title translate>` enough? That would be best to show as an example, if correct.

Reply:
I verified that it actually is, so I changed it accordingly.